src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Onramps style guide
+contentType: page
+template: basicDoc
+topics:
+  - onramps
+---
+
+import apmOnrampsIssuesExample from 'images/apm_screenshot-crop_onramps-issues-example.webp'
+
+import apmActiveEmptyState from 'images/apm_screenshot-crop_active-empty-state.webp'
+
+Onramps content is information that appears our UI when a user clicks the **?** icon. This content is meant to be truncated and contextualized to where the user is and what they need to know to perform a task.
+
+## How should I name the section titles?
+
+When applicable, use an FAQ format where the headers are in question format.
+
+For example, here's the Issues tile with headers formatted into questions.
+
+<img
+    title=""
+    alt=""
+    src={apmOnrampsIssuesExample}
+    style={{ maxWidth: '50%' }}
+/>
+
+
+If FAQ format is not applicable, use our standard headers for titles.
+
+## What if a tile has two states (active and empty)?
+
+A tile is something that's clickable and may or may not contain data, depending on if customers have set it up. This is called an active vs. an empty state, and it's something to consider when writing onramps content. If the tile has no data, it's in empty state, and if the user clicks on it, they may be directed to a set up page. If it has data, it's in active state, and our onramps content will want to help users interpret the data.
+
+For example, here's the Service levels tile in active and empty states. The onramps content should contain info that's helpful for both states.
+
+<img
+    title=""
+    alt=""
+    src={apmActiveEmptyState}
+    style={{ maxWidth: '75%' }}
+/>
+
+## How long should the content be?
+
+If you're not using collapsers, **keep the character count under 1200** for the entire onramps section.
+
+## How should I bold/italicize to emphasize something?
+
+Since we can't add code formatting to onramps content, use bold and italics to emphasize something as needed.
+
+## How much detail should I include?
+
+If the onramps content isn't in the main place where the feature is (i.e., onramps content for the Service levels tile on the APM Summary page), then it’s not the place to include a lot of detail.
+
+## Should I add links out to docs?
+
+It's okay to add a link to docs "for more details" on something, but here are some considerations:
+* Add a docs link if there's relevant information and it's too long to summarize/explain in onramps. 
+* The onramps content will persist after the user clicks on something, so add a docs link if it's relevant to where the user might go next (i.e., if you click on the Issues tile and are taken to the **Issues** page, you'll still see the onramps popup on the same, so you may want to learn how to configure issue thresholds on this page).
```

Comment:
```suggestion
* The onramps content will persist after the user clicks on something, so add a docs link if it's relevant to where the user might go next (i.e., if you click on the Issues tile and are taken to the **Issues** page, you'll still see the onramps popup on the same page, so you may want to learn how to configure issue thresholds on this page).

Avoid making a link to docs the sole content of an onramp.  When linking, use the established UI style for docs links. Text is "See our docs" and use the external link icon. 
```

Reply:
regarding Michelle's comment above, the external link icon she mentions works automatically for links so we don't have to mention that apparently.